normandy/base/middleware.py

```diff
@@ -40,7 +41,30 @@ class RequestSummaryLogger(MiddlewareMixin, OriginalRequestSummaryLogger):
     """
     Adapt mozilla_cloud_services_logger's request logger to Django 1.10 new-style middleware.
     """
-    pass
+
+    # TODO: Remove this once it is fixed upstream
```

Comment:
This is unlikely to get fixed upstream as the project has been abandoned. So I filed: https://github.com/mozilla/normandy/issues/1334

Reply:
Lets add the issue to the to the comment here.